test/smoke/features/cli/cli_deploy_springboot_service.feature

```diff
@@ -0,0 +1,33 @@
+Feature: CLI: Deploy Spring Boot service
+
+  Scenario: CLI deploy drools-quarkus-example service
+    Given Kogito Operator is deployed
+
+    When CLI deploy spring boot example service "jbpm-springboot-example"
+
+    Then Kogito application "jbpm-springboot-example" has 1 pods running within 10 minutes
+    And HTTP GET request on service "jbpm-springboot-example" with path "orders" is successful within 2 minutes
+
+#####
+
+  Scenario: CLI deploy jbpm-springboot-example service with persistence
+    Given Kogito Operator is deployed with dependencies
+    And CLI deploy spring boot example service "jbpm-springboot-example" with persistence enabled
+    And Kogito application "jbpm-springboot-example" has 1 pods running within 10 minutes
+    And HTTP GET request on service "jbpm-springboot-example" with path "orders" is successful within 3 minutes
```

Comment:
I suppose that this step is to make sure that route is up and running, right?

Reply:
yes. it does check the route and check application is answering